api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2304,15 +2304,15 @@ public void removeObject(String bucketName, String objectName)
    * objectNames.add("my-objectname1");
    * objectNames.add("my-objectname2");
    * objectNames.add("my-objectname3");
-   * for (Result<DeleteError> errorResult: minioClient.removeObject("my-bucketname", objectNames)) {
+   * for (Result<DeleteError> errorResult: minioClient.removeObjects("my-bucketname", objectNames)) {
    *     DeleteError error = errorResult.get();
    *     System.out.println("Failed to remove '" + error.objectName() + "'. Error:" + error.message());
    * } }</pre>
    *
    * @param bucketName Bucket name.
    * @param objectNames List of Object names in the bucket.
    */
-  public Iterable<Result<DeleteError>> removeObject(final String bucketName, final Iterable<String> objectNames) {
+  public Iterable<Result<DeleteError>> removeObjects(final String bucketName, final Iterable<String> objectNames) {
```

Comment:
Is it agreed to change the name?  If so, its a breaking change and you would need to bump up major version.

Reply:
I think we should adopt the same design decision to have 2 remove apis, one for single object and one for multiple objects. All other SDK's, Python, JavaScript, Go and .Net have 2 separate apis: `removeObject` and `RemoveObjects`.
We should do the same for Java SDK too.

Reply:
>Is it agreed to change the name?
Checked with @kannappanr @deekoder  before changing the name. 
Only the public method name is changed. Made the changed in `minio-java` . Please tell what can break because of this? I can retrofit it in the PR.

Reply:
Only the public method name is changed. Made the changed in `minio-java` . Please tell what can break because of this? I can retrofit it in the PR.

Reply:
We did discuss the impact of this breaking change with AB. I think we just have to call it out as a BREAKING CHANGE when we do a release with this PR.